cylc/flow/id.py

```diff
@@ -699,6 +699,16 @@ def detokenise(
     selectors: bool = False,
     relative: bool = False,
 ) -> str:
+    return '/'.join(
+        detokenise_to_list(tokens, selectors, relative)
+    )
+
+
+def detokenise_to_list(
```

Comment:
You might find the functionality you're looking for here already exists on the `Tokens` object.

Reply:
I think I checked at the time ... but I'll double-check.

Reply:
... You're right, done. Thanks.